src/pages/workspace/companyCards/addNew/CardTypeStep.tsx

```diff
@@ -0,0 +1,127 @@
+                keyForList: CONST.COMPANY_CARDS.CARD_TYPE.AMEX,
+                isSelected: typeSelected === CONST.COMPANY_CARDS.CARD_TYPE.AMEX,
+                leftElement: (
+                    <Icon
+                        src={Illustrations.AmexCompanyCards}
+                        height={variables.iconSizeExtraLarge}
+                        width={variables.iconSizeExtraLarge}
+                        additionalStyles={styles.mr3}
+                    />
+                ),
+            },
+            {
+                value: CONST.COMPANY_CARDS.CARD_TYPE.MASTERCARD,
+                text: translate('workspace.companyCards.addNewCard.cardProviders.mastercard'),
+                keyForList: CONST.COMPANY_CARDS.CARD_TYPE.MASTERCARD,
+                isSelected: typeSelected === CONST.COMPANY_CARDS.CARD_TYPE.MASTERCARD,
+                leftElement: (
+                    <Icon
+                        src={Illustrations.MasterCardCompanyCards}
+                        height={variables.iconSizeExtraLarge}
+                        width={variables.iconSizeExtraLarge}
+                        additionalStyles={styles.mr3}
+                    />
+                ),
+            },
+            {
+                value: CONST.COMPANY_CARDS.CARD_TYPE.VISA,
+                text: translate('workspace.companyCards.addNewCard.cardProviders.visa'),
+                keyForList: CONST.COMPANY_CARDS.CARD_TYPE.VISA,
+                isSelected: typeSelected === CONST.COMPANY_CARDS.CARD_TYPE.VISA,
+                leftElement: (
+                    <Icon
+                        src={Illustrations.VisaCompanyCards}
+                        height={variables.iconSizeExtraLarge}
+                        width={variables.iconSizeExtraLarge}
+                        additionalStyles={styles.mr3}
+                    />
+                ),
+            },
+        ],
+        [styles.mr3, translate, typeSelected],
+    );
+
+    return (
+        <ScreenWrapper
+            testID={CardTypeStep.displayName}
+            includeSafeAreaPaddingBottom={false}
+            shouldEnablePickerAvoiding={false}
+            shouldEnableMaxHeight
+        >
+            <HeaderWithBackButton
+                title={translate('workspace.companyCards.addCardFeed')}
+                onBackButtonPress={handleBackButtonPress}
+            />
+
+            <Text style={[styles.textHeadlineLineHeightXXL, styles.ph5, styles.mv3]}>{translate('workspace.companyCards.addNewCard.yourCardProvider')}</Text>
+            <SelectionList
+                ListItem={RadioListItem}
+                onSelectRow={({value}) => setTypeSelected(value)}
+                sections={[{data}]}
+                shouldSingleExecuteRowSelect
+                initiallyFocusedOptionKey={addNewCard?.data.cardType}
+                shouldUpdateFocusedIndex
+            />
+            <Button
+                success
+                isDisabled={!typeSelected}
```

Comment:
Sorry @waterim, instead disabling button we need to show the error message as per figma https://www.figma.com/design/3QyVvvVn4it51jbSDcDotW/%23wave-control%3A-Workspace-Third-Party-Card-Feeds-V1?node-id=4845-22360&node-type=FRAME&t=oPOE6Hg3UeS3dPZt-0

Ref comp - https://github.com/Expensify/App/blob/main/src/pages/workspace/accounting/netsuite/import/NetSuiteImportCustomFieldNew/substeps/ChooseSegmentTypeStep.tsx#L48-L70